src/main/java/microsoft/exchange/webservices/data/HttpClientWebRequest.java

```diff
@@ -127,6 +138,14 @@ public void prepareConnection() throws EWSHttpException {
       rcBuilder.setConnectTimeout(getTimeout());
       rcBuilder.setRedirectsEnabled(isAllowAutoRedirect());
       rcBuilder.setSocketTimeout(getTimeout());
+	  
+	  // fix issue #144 + #160: if we used NTCredentials from above: these are NT credentials
+      if (getUserName() != null) {
+    	ArrayList<String> authPrefs = new ArrayList<String>();
+    	authPrefs.add(AuthSchemes.NTLM);
+    	rcBuilder.setTargetPreferredAuthSchemes(authPrefs).build();
+      }
+      //
```

Comment:
* should this behavior also be added to `prepareAsyncConnection`? 
* `rcBuilder.build()` is called twice ref. [l146](https://github.com/rlambrechts/ews-java-api/blob/master/src/main/java/microsoft/exchange/webservices/data/HttpClientWebRequest.java#L146) / [l149](https://github.com/rlambrechts/ews-java-api/blob/master/src/main/java/microsoft/exchange/webservices/data/HttpClientWebRequest.java#L149) is this usefull?
* please align the comment in [l142](https://github.com/rlambrechts/ews-java-api/blob/master/src/main/java/microsoft/exchange/webservices/data/HttpClientWebRequest.java#L142)